tools/cdk-release/test/conventional-commits.test.ts

```diff
@@ -39,38 +39,69 @@ describe('getConventionalCommitsFromGitHistory', () => {
   });
 });
 
-// NOTE - These test currently use real package.json data to determine package's stability.
 describe('filterCommits', () => {
   const commits: ConventionalCommit[] = [
-    buildCommit({
-      type: 'feat',
-      scope: 'scope',
-      subject: 'super important feature',
-    }),
-    buildCommit({
-      type: 'fix',
-      scope: 'example-construct-library', // really hope we don't stabilize this one
-      subject: 'hairy bugfix',
-      notes: [
-        {
-          title: 'BREAKING CHANGE',
-          text: 'this is a breaking change',
-        },
-      ],
-    }),
+    commitWithScope('aws-stable'),
+    commitWithScope('aws-experimental'),
+    commitWithScope(),
```

Comment:
This reads weirdly... the function is called `commitWithScope()`, and yet returns a commit without a scope...?

How about making `scope` a required argument of `commitWithScope()`, and have a separate function `commitWithoutScope()`?

Reply:
Blergh. Would you like it better as `commitWithScope('')` or `commitWithScope(undefined)` to be more explicit?